### PR TITLE
add language course

### PR DESCRIPTION
**Pull request by NgocVu1994**
thêm language cho course

### CODE REVIEW

**Comment by thiennv57** [comment]
OK

File: app/views/admin/courses/_course.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <tr>
   <td class="name"><%= link_to course.name, admin_course_path(course) %></td>
-  <td class="avatar"><%= image_course_tag(course, course.language) %></td>
+  <td class="avatar"><%= image_course_tag(course, course.programming_language_name) %></td>
```
Comment (truonglocbinh) [inline app/views/admin/courses/_course.html.erb]: remove ()

**Comment by thao33** [comment]
Ok.

**Comment by truonglocbinh** [comment]
ok

**Pull request merged**